### PR TITLE
Numberinput

### PR DESCRIPTION
**Pull request by HerrTopi**

### CODE REVIEW

**Comment by ke1k0** [comment]
When deprecating props in the component, please indicate it as a breaking change in the commit, so it appears in the change log :)

File: packages/ui-number-input/src/NumberInput/styles.js [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+            fontSize: componentTheme.largeFontSize,
+            height: componentTheme.largeHeight
+          })
+    },
+    input: {
+      label: 'numberInput_input',
+      all: 'initial',
+      textAlign: 'start',
+      direction: 'inherit',
+      '&::-ms-clear': { display: 'none' },
```
Comment (matyasf) [inline packages/ui-number-input/src/NumberInput/styles.js:146]: this is no longer needed in the browsers we support, see https://developer.mozilla.org/en-US/docs/Archive/Web/CSS/::-ms-clear

File: packages/ui-number-input/src/NumberInput/styles.css [hunk 2]
```diff
@@ -1,138 +0,0 @@
...
-  &.large {
-    font-size: var(--largeFontSize);
-    height: var(--largeHeight);
-  }
-}
-
-/* stylelint-disable selector-max-type, no-descending-specificity */
-
-.input,
-input[type].input {
```
Comment (matyasf) [inline packages/ui-number-input/src/NumberInput/styles.css:69]: I'm just curious, what purpose did this serve? Did it need something to migrate to the new code?

**Review by matyasf — APPROVED** [review]

**Comment by ke1k0** [comment]
The refact branch is merged into next, so please make the modifications to match the new componentTheme handling: https://github.com/instructure/instructure-ui/pull/316.

**Review by Brailor — APPROVED** [review]

File: packages/ui-number-input/src/NumberInput/styles.css (hunk 2 above)
Comment (HerrTopi) [reply packages/ui-number-input/src/NumberInput/styles.css:69]: I think with the restructuring of the styling, this line is implied in the style.

**Pull request closed without merging**